### DISCUSSION TITLE
How to enable the trace for Trino jdbc driver?

### DISCUSSION THREAD

**Discussion by srsooraj82** [trinodb/trino#22896] [Q&A]
Hi,

Any Idea, how can we enable a trace for the trino  Jdbc driver to  get some trace logs to debug  ? I could not find anything in the documentation.

Regards,
srs

**Comment by nineinchnick** [comment]
You need to make sure the `GlobalOpenTelemetry` is set. The driver tries to get it here: https://github.com/trinodb/trino/blob/master/client/trino-jdbc/src/main/java/io/trino/jdbc/NonRegisteringTrinoDriver.java#L85

If that's configured, the OpenTelemetry JDBC instrumentation will create spans for any active trace, or create a new trace. The trace will be propagated to the Trino server, so if you have tracing configured there too, you'll be able to see the whole query execution.

@electrum should we document this?

**Reply by electrum** [reply]
This would be good to document.
